src/test/app/NFToken_test.cpp

```diff
@@ -3927,31 +3927,33 @@ class NFToken_test : public beast::unit_test::suite
             env.close();
         }
 
-        // Lambda to set the balance of all passed in accounts to gwXAU(1000).
-        auto setXAUBalance_1000 =
+        // Lambda to set the balance of all passed in accounts to
+        // gwXAU(amount)
+        auto setXAUBalance =
             [this, &gw, &gwXAU, &env](
                 std::initializer_list<std::reference_wrapper<Account const>>
                     accounts,
-                int line) {
+                int line,
+                int amount = 1000) {
```

Comment:
I'm not wild about the default argument.  If I read the test's code, `setXAUBalance` no longer tells me what the balance is being set to.  And it makes a difference in each of the tests.  But since this is unit test code this is not a hill I'm ready to die on.

There are two approaches that would be better in my opinion.

1. You could leave the code as it was and have `broker` pay `gwXAU(500)` to `gw` shortly after calling `setXAUBalance_1000`.  Then `broker` would have the necessary lower balance and the old interface would be left alone.

2. You could make the `amount` mandatory. This would be more editing because you'd need to add `1000` to almost every call to `setXAUBalance`.  But it's probably the cleaner solution.  Also, if you follow this approach, you should swap the order of the `line` and `amount` arguments.  The `line` argument is just, uh, line noise (:smile:) when reading the code.  It should go last.

But, again, I'll not block the pull request for this particular change.